rust-toolchain

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2021-01-13"
+channel = "nightly-2021-04-18"
```

Comment:
Was this on purpose?

Reply:
Good catch, I think this was left over from something else I was working on.

Reply:
Reverted in d49aabc